safe/definitions/hazard_classifications.py

```diff
@@ -123,9 +135,9 @@
         exposure_structure
     ]
 }
-
-earthquake_mmi_hazard_classes = {
-    'key': 'earthquake_mmi_hazard_classes',
+# do not use these old classes until we can trace the origin
+earthquake_mmi_old_hazard_classes = {
+    'key': 'earthquake_mmi_old_hazard_classes',
```

Comment:
+1 to keep the old key - we can add remove it from hazard types if we want to disabled it.